media/js/firefox/features/picture-in-picture.js

```diff
@@ -0,0 +1,101 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* global YT */
+/* eslint no-unused-vars: [2, { "varsIgnorePattern": "onYouTubeIframeAPIReady" }] */
+
+// YouTube API hook has to be in global scope
+function onYouTubeIframeAPIReady() {
+    'use strict';
+
+    // Play the video only once the API is ready.
+    Mozilla.pipVideoPlay();
+}
+
+(function() {
+    'use strict';
+
+    var videoLink = document.querySelector('.js-video-play');
+    var src = 'https://www.youtube.com/iframe_api';
+
+    function loadScript() {
+        var tag = document.createElement('script');
+        tag.src = src;
+        var firstScriptTag = document.getElementsByTagName('script')[0];
+        firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
+    }
+
+    function isScriptLoaded() {
+        return document.querySelector('script[src="' + src + '"]') ? true : false;
+    }
+
+    function playVideo() {
+        var videoId = videoLink.getAttribute('data-id');
+        var title = videoLink.getAttribute('data-video-title');
+
+        new YT.Player(videoLink, {
+            width: 500,
+            height: 281,
+            videoId: videoId,
+            playerVars: {
+                modestbranding: 1, // hide YouTube logo.
+                rel: 0, // do not show related videos on end.
```

Comment:
FYI: Still worth adding here, as it isolated videos to just our channel, but this tag behavior has been changed _(in relation to the comment after the attribute)_ in [Aug 2018](https://developers.google.com/youtube/player_parameters#august-23,-2018).